automation/taskcluster/androidTest/flank-x86.yml

```diff
@@ -38,7 +38,7 @@ gcloud:
   performance-metrics: true
 
   test-targets:
-   - class org.mozilla.fenix.ui#SmokeTest
+   - package org.mozilla.fenix.ui#SmokeTest
```

Comment:
If I'm not mistaken, at least how I run them locally from command line, this should be:
- class org.mozilla.fenix.ui.SmokeTest